controllers/__init__.py

```diff
@@ -16,10 +16,13 @@
 from .user_type import user_type_bp
 from .tenancy_config import tenancy_config_bp
 from .tutorial_quiz import tutorial_quiz_bp
-from .email import email_bp
+from .email_controllers import email_bp
```

Comment:
Why is this being renamed?

Reply:
i think i  didin't change anything about these two line.

Reply:
Ok, they got changed somehow, so you'll need to change them back. I think you should be able to just change that line to `from .email import email_bp` and that'll fix it